### PR TITLE
PS-8926 8.1.0-1 release notes and release tasks (8.1 innovation)

### PR DESCRIPTION
**Pull request by patrickbirch**
deleted:    docs/release-notes-index.md
	new file:   docs/release-notes/8.1.0-1.md
	new file:   docs/release-notes/release-notes-index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.1.0-1.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Percona Server for MySQL implements telemetry that fills in the gaps in our understanding of how you use Percona Server to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. Find more information in the [Telemetry on Percona Server for MySQL](../telemetry.md) document.
+
+The following components are added:
+
+* `keyring_vault`
+* `masking_functions`
+* `binlog_utils`
+* `percona-udf`
+* `audit_log`
```
Comment (percona-ysorokin) [inline docs/release-notes/8.1.0-1.md]:
```suggestion
* `component_keyring_vault`
* `component_masking_functions`
* `component_binlog_utils`
* `component_percona_udf`
* `component_audit_log_filter`
```

Comment (percona-ysorokin) [reply docs/release-notes/8.1.0-1.md]: I would also regroup this a bit.
```
component_masking_functions
component_binlog_utils
component_percona_udf
```
are fine here (simplification with INSTALL COMPONENT is the key point)

Comment (percona-ysorokin) [reply docs/release-notes/8.1.0-1.md]: At the same time `component_keyring_vault ` and `component_audit_log_filter` deserve individual paragraphs (please see the Engineering meeting presentation for more details).

Comment (percona-ysorokin) [reply docs/release-notes/8.1.0-1.md]: Also, a paragraph about Percona MTR test cases reorganizations would also be nice (the same presentation).

File: docs/release-notes/8.1.0-1.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* [PS-8932] : The documentation mentioned `ALL_O_DIRECT` parameter for Percona Server for MySQL 8.x but flag was removed.
+* [PS-8949] : The MyRocks configuration failed on a Mac with M1.
+
+## Deprecation or removal
+
+* The `mysql_native_password` authentication plugin is deprecated and subject to removal in a future version.
+* The TokuDB storage engine is removed.
+* The MyRocks ZenFS support is removed.
+* The `libreadline` dependency library (used for command-line clients to maintain command history) is removed.
+* The `keyring_value` plugin is removed.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.1.0-1.md]:
```suggestion
* The `keyring_vault` plugin is removed.
```

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.1.0-1.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    * RocksDB MTR changes are reverted
+* Travis CI integration
+* Supporting `readline` as a alternative to editline library is removed.
+* The `audit_log` (audit version 1) plugin is removed
+* The "include/ext" pre-C++17 compatibility headers are removed.
+* The `keyring_vault` plugin is removed.
+* The `data_masking` UDF shared object (.so) is removed.
+* The `binlog_utils_udf` UDF shared object (.so) is removed.
+* The `percona_udf` UDF shared object (.so) is removed.
+* The `audit_log` v1 plugin is removed.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.1.0-1.md]: Already in the list

**Review by percona-ysorokin — APPROVED** [review]
LGTM with one comment addressed

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by eleo007 — APPROVED** [review]

**Pull request merged**